content/news/2023.pml

```diff
@@ -54,6 +54,26 @@ Older News: <%=reflist%>
 </div>
 -->
 
+
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="nio-skyos">
+  <a href="#nio-skyos"><span class="tag-span"># </span></a>
+    24 November 2023: NIO announces seL4-based SkyOS operating system
+</div>
+<div class="news-text-container">
+     <img src="/Foundation/Membership/LOGOS/NIO.svg"
+   style="width:200px; padding-left:1.5em; float:right"
+   alt="nio">
+  <p>
+    We are excited to share that <a href="https://nio.com/">NIO</a> recently unveiled their seL4-based SkyOS operating system, designed for Software Defined Vehicles, which they have been working on relentlessly for the past two years. At the <a href="/Foundation/Summit/2023">seL4 Summit 2023</a> Qiyan Wang, NIO’s Global VP of Digital Systems, announced that NIO cars based on seL4 are planned for next year!
+  </p>
```

Comment:
I suggest:
<a href="https://www.linkedin.com/posts/qiyan-wang-8b0b2145_skyos-was-unveiled-to-the-public-for-the-activity-7110702765789642752-Iys0">seL4-based SkyOS operating system</a>